.eslintrc

```diff
@@ -255,7 +255,8 @@
         "prettier/prettier": [
             "error",
             {
-                "endOfLine": "auto"
+                "endOfLine": "auto",
+                "quoteProps": "preserve"
```

Comment:
This PR shouldn't change central settings and only focus on the `ui-components` module. The central changes are reserved for PR #689

Reply:
@tobiasqueck, problem is, we have conflicting eslint settings in `main`-branch, which means I can't solve the eslint issues in `ui-components` without resolving this conflict first. For background information about the issue with conflicting rules see https://github.com/SAP/open-ux-tools/pull/689#issuecomment-1255628784 and https://github.com/SAP/open-ux-tools/pull/689#issuecomment-1256112736.

If we do not fix it here, we need either to add a `eslint-disable` comment to the code and remove it after the change has done, or, which I think is the better approach, I can create a separate PR with just the fix for conflicting eslint rules.

Reply:
-> https://github.com/SAP/open-ux-tools/pull/701